src/locale/locales/fr/messages.po

```diff
@@ -1540,11 +1462,11 @@ msgstr "Les fils d’actu sont des algorithmes personnalisés qui se construisen
 
 #: src/screens/Onboarding/StepTopicalFeeds.tsx:76
 msgid "Feeds can be topical as well!"
-msgstr ""
+msgstr "Les fils d’actu peuvent également être thématique !"
```

Comment:
Accord (`thématiques` is used as an adjective, so it should be accorded with `Les fils d’actu`) + non-breaking space.

```suggestion
msgstr "Les fils d’actu peuvent également être thématiques !"
```